requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.14.3
 pytest>=3.6.1
-requests>=2.20.0
\ No newline at end of file
+requests>=2.20.0
+pandas>=0.23.4
```

Comment:
@nikhiln could we achieve the same without needing pandas? I'd like to keep dependencies to a minimum if possible.

Reply:
@zermelozf We can, but we would need to then extract csv and parse it. Pandas does that conveniently. Aren't we using the pandas for data science part later on?

Reply:
@zermelozf Replaced it with standard csv lib.